pkg/kubelet/kuberuntime/kuberuntime_manager_test.go

```diff
@@ -732,6 +733,7 @@ func TestComputePodActions(t *testing.T) {
 		mutatePodFn    func(*v1.Pod)
 		mutateStatusFn func(*kubecontainer.PodStatus)
 		actions        podActions
+		resetStatusFn  func(*kubecontainer.PodStatus)
```

Comment:
Can we `createTestRuntimeManager` inside the loop? So that we don't need to reset the status.

Reply:
Makes sense... maybe it could go into another PR?